spotbugs-exclude.xml

```diff
@@ -35,4 +35,8 @@
         <Method name="poll"/>
         <Bug pattern="RV_RETURN_VALUE_IGNORED_BAD_PRACTICE" />
     </Match>
+    <Match>
+        <Class name="io.ballerina.stdlib.kafka.plugin.CodeSnippetGenerationCodeActionTest" />
+        <Bug pattern="UPM_UNCALLED_PRIVATE_METHOD" />
+    </Match>
```

Comment:
As we have removed the `@DataProvider`, I think we can remove this part as well.

Reply:
Removed in https://github.com/ballerina-platform/module-ballerinax-kafka/pull/362/commits/73b6b7eb1f8d9bffed690ddcc2c17838efd7f9ed